src/components/VideosList/VideosList.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import { items } from './youtube-videos-mock.json';
+
+const { snippet: channel } = items.shift();
+
+const VideoCard = styled.div`
+  max-width: 320px;
+  margin: 0.75rem;
+  p {
+    margin: 0;
+    color: #555555;
+  }
+  .channel-avatar {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+  }
+  .margin-x {
+    margin: 0 0.75rem;
+  }
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: flex-start;
+`;
+
+const Link = styled.a`
+  color: #000000;
+  font-weight: 700;
+`;
+
+const VideosList = () =>
+  items.map(({ id, snippet }) => (
+    <VideoCard key={id.videoId}>
+      <figure>
+        <img src={snippet.thumbnails.medium.url} alt={`${snippet.title} thumbnail`} />
+      </figure>
+      <Wrapper>
+        <img
+          src={channel.thumbnails.default.url}
+          alt={channel.channelTitle}
+          className="channel-avatar"
+        />
+        <div className="margin-x">
```

Comment:
Same suggestion as in the ToggleSwitch component, these classes can be their own styled components. It's not "over-using" the styled components even if it feels like you are creating one for minor styling details. It's a good practice and very useful on bigger projects where the styles may be on a different file and having nested styles/classes can become difficult to debug in some cases. You will find the styled components approach is easier to maintain in the long run.